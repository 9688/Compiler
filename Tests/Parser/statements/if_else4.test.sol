
IF
     list
          x
     BLOCK
          list
               "@++"
                    x
     list
          "@--"
               x

