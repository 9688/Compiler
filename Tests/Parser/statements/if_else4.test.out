юдд''
   юдд'if'
      цдд'x'
      цдд'block'
      Ё  юдд'@++'
      Ё     юдд'x'
      юдд'@--'
         юдд'x'

